docs/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Flexy OSIsoft Changelog
 
+## V2.8
+### Major Changes
++ None
+### Minor Changes
++ Fixed malformed post requests for piwebapi calls

```

Comment:
Weren't there GETs and PUTs that were fixed as well?

Reply:
fixed

Reply:
The GETs and PUTs change was not performed in a library. IT was done within the OSISoft connector at https://github.com/hms-networks/flexy-osisoft-connector/pull/91/files

Reply:
The library update is a separate change from a few weeks ago

Reply:
What are you referring to @alexjhawk?

Reply:
@TomKimsey was your original comment not referring to the GETs and PUTs that were changed in https://github.com/hms-networks/flexy-osisoft-connector/pull/91/files?

Reply:
I never implied it was performed in a library.